### ISSUE TITLE
Color Contrast: Poll Footer

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#173]
**Describe the bug**
Element has insufficient color contrast of 2.99 (foreground color: `#888a8b`, background color: `#eceff1`, font size: 10.5pt (14px), font weight: normal). 

**To Reproduce**
Element location: `.poll-footer > p`
Element source: `<p>67 votes · 3 weeks left</p>`

**Expected behavior**
Expected contrast ratio of 4.5:1

Possible fix:
- remove the additional `opacity: .64` causing the 6.81:1 ratio to slip to less than 3:1

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll update you soon.
